lib/rouge/lexers/elixir.rb

```diff
@@ -120,7 +120,11 @@ class Elixir < RegexLexer
               rule %r/[\\#]/, toktype
             end
 
-            uniq_chars = "#{open}#{close}".squeeze
+            uniq_chars = if open == close
+                           open.squeeze
```

Comment:
I think that the point of squeeze was to attempt to remove duplicate characters, but it didn't take into account escaping.
With the condition being explicit now, both `squeeze`s can be removed.